frontend/packages/dev-console/locales/en/dev-console.json

```diff
@@ -0,0 +1,3 @@
+{
+  "View metrics for {{title}}": "View metrics for {{title}}"
```

Comment:
Something isn't right here... We should make sure we're using the same namespace as the rest of dev console. We could consider removing this from your PR since it's really code owned by the dev console team.